crates/lib/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+pub mod filters;
+pub mod partials;
```

Comment:
I'm sorry, I missed this.  partials should be in `liquid_core`.  Doing tis should resolve the compile error the CI is getting with `--no-default-features`

Reply:
No bother, I'll do it this evening :)

Reply:
Shall I fix clippy? There's quite a lot of them (maybe because I updated to 1.42?)

Reply:
As much as you want.  If needed, I can come in and fix other clippy lints

Reply:
I'll do my max then :D